test/test_api/test_api.py

```diff
@@ -15,22 +18,105 @@
 from sklearn.base import clone
 from sklearn.ensemble import VotingClassifier, VotingRegressor
 
-from smac.runhistory.runhistory import RunHistory
+from smac.runhistory.runhistory import RunHistory, StatusType
 
 import torch
 
 from autoPyTorch.api.tabular_classification import TabularClassificationTask
 from autoPyTorch.api.tabular_regression import TabularRegressionTask
+from autoPyTorch.data.tabular_validator import TabularInputValidator
 from autoPyTorch.datasets.resampling_strategy import (
     CrossValTypes,
     HoldoutValTypes,
 )
+from autoPyTorch.datasets.tabular_dataset import TabularDataset
+from autoPyTorch.evaluation.train_evaluator import TrainEvaluator
 from autoPyTorch.optimizer.smbo import AutoMLSMBO
 from autoPyTorch.pipeline.components.training.metrics.metrics import accuracy
 
+from api_utils import dummy_traditional_classification  # noqa (E402: module level import not   at top of file)
```

Comment:
Rather than having the noqa, isn't it better to put it in the correct location?

Reply:
I thought its a test so didnt care, but surely I can do that